From db40ec43617c802b5dff4af43c095924bea17ac6 Mon Sep 17 00:00:00 2001
From: Miroslav Rezanina <mrezanin@redhat.com>
Date: Thu, 7 Jun 2018 07:43:09 +0200
Subject: [PATCH 01/15] Disable aarch64 devices reappeared after 2.12 rebase

RH-Author: Miroslav Rezanina <mrezanin@redhat.com>
Message-id: <f72359b74f5219ef13ae074e11938a97c1314200.1528355911.git.mrezanin@redhat.com>
Patchwork-id: 80592
O-Subject: [RHEL-7.6 qemu-kvm-rhev PATCH 1/7] Disable aarch64 devices reappeared after 2.12 rebase
Bugzilla: 1586357
RH-Acked-by: Thomas Huth <thuth@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Laurent Vivier <lvivier@redhat.com>

From: Miroslav Rezanina <mrezanin@redhat.com>

Up to QEMU 2.11 we disabled aarch64 devices by removing files from
Makefile.
As default_config options were added, we use them to disable devices
since 2.12. However, we can't turn CONFIG_ARM_V7M (turning it off will
break build) so we have to manually remove armv7m module from Makefile.

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 hw/arm/Makefile.objs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/arm/Makefile.objs b/hw/arm/Makefile.objs
index 2885e3e..3e8b167 100644
--- a/hw/arm/Makefile.objs
+++ b/hw/arm/Makefile.objs
@@ -16,7 +16,7 @@ obj-$(CONFIG_STRONGARM) += collie.o
 obj-$(CONFIG_VERSATILE) += vexpress.o versatilepb.o
 obj-$(CONFIG_ZYNQ) += xilinx_zynq.o
 
-obj-$(CONFIG_ARM_V7M) += armv7m.o
+#obj-$(CONFIG_ARM_V7M) += armv7m.o
 obj-$(CONFIG_EXYNOS4) += exynos4210.o
 obj-$(CONFIG_PXA2XX) += pxa2xx.o pxa2xx_gpio.o pxa2xx_pic.o
 obj-$(CONFIG_DIGIC) += digic.o
-- 
1.8.3.1

